plugins/webp-uploads/hooks.php

```diff
@@ -534,22 +534,26 @@ function webp_uploads_update_image_references( $content ): string {
 
 	// This content does not have any tag on it, move forward.
 	// TODO: Eventually this should use the HTML API to parse out the image tags and then update them.
-	if ( ! preg_match_all( '/<(img)\s[^>]+>/', $content, $img_tags, PREG_SET_ORDER ) ) {
+	if ( 0 === (int) preg_match_all( '/<(img)\s[^>]+>/', $content, $img_tags, PREG_SET_ORDER ) ) {
```

Comment:
The use of `(int)` isn't needed here either...

Reply:
Well, actually it is. Because it's `preg_match_all()` so we don't know the number of image tags being matched. Otherwise, it could be `1 === ...`. So by casting the return value to `(int)`, it will convert both `0` and `false` to `0`. Alternatively, this could be:

```suggestion
	if ( false === (bool) preg_match_all( '/<(img)\s[^>]+>/', $content, $img_tags, PREG_SET_ORDER ) ) {
```

But `false` indicates the error state. The integer return value is the normal case, the number of matches.